Firestore/Source/Public/FirebaseFirestore/FIRCollectionReference.h

```diff
@@ -29,8 +29,8 @@ NS_ASSUME_NONNULL_BEGIN
 NS_SWIFT_NAME(CollectionReference)
 @interface FIRCollectionReference : FIRQuery
 
-/** :nodoc: */
-- (id)init __attribute__((unavailable("FIRCollectionReference cannot be created directly.")));
+/** Initialiser. Only for use in unit tests. */
```

Comment:
Please use American English in code and comments. "Initializer". I'm ok with British English in PR descriptions though.